cmd/node/config/config.toml

```diff
@@ -0,0 +1,19 @@
+[General]
```

Comment:
where do we define the public keys list?

Reply:
for poc i've just added the option to search the nodes by the identifier field, will add the option to define also public keys